homeassistant/components/seventeentrack/__init__.py

```diff
@@ -124,13 +129,73 @@ async def get_packages(call: ServiceCall) -> ServiceResponse:
             ]
         }
 
+    async def add_package(call: ServiceCall) -> ServiceResponse:
+        config_entry_id = call.data[ATTR_CONFIG_ENTRY_ID]
+        tracking_number = call.data.get(ATTR_TRACKING_NUMBER)
+        friendly_name = call.data.get(ATTR_INFO_TEXT, "")
+
+        config_entry: ConfigEntry | None = hass.config_entries.async_get_entry(
+            config_entry_id
+        )
```

Comment:
When if the config entry given isn't found?
The current typing suggests that it isn't possible.

(I wonder why the typing was overridden in this case)

Reply:
That's a good question, I took this from the `get_packages` function above. I'll do some more investigation.